app/models/grantee.rb

```diff
@@ -11,43 +11,56 @@ def hses_link
     links[:html]
   end
 
-  def centers_total
-    attributes[:numberOfCenters]
+  def centers
+    current_grant&.centers
   end
 
   def complaints_per_fy
-    attributes[:totalComplaintsFiscalYear]
+    grants.map(&:complaints_per_fiscal_year).each_with_object({}) do |count, obj|
+      obj.merge!(count) { |k, v1, v2| v1 + v2 }
+    end.sort.reverse.to_h
+  end
+
+  def current_grant
+    grants.first
   end
 
   def name
-    attributes[:name]
+    attributes[:granteeName]
   end
 
   def region
-    attributes[:region]
+    current_grant&.region
   end
 
   def complaints
-    relationships[:issues][:data].map do |complaint_hash|
-      Complaint.new(complaint_hash)
+    relationships[:issues][:data].map do |complaint_data|
+      Complaint.new(complaint_data)
     end
   end
 
   private
 
+  def grants
+    @grants ||= data[:attributes][:grants]
+      .map { |grant| Grant.new(grant) }
+      .sort_by(&:start_date)
```

Comment:
it's unclear to me if program start date or program end date is a better indicator of the currently "active" grant, but decided to go with start date (and it's easy to swap out when/if we find out different)

Reply:
I believe that the current format is that everything returned is active. We had talked with Uira about also including the precending grant for any active grants so that might need to change slightly